tests/baselines/reference/looseThisTypeInFunctions.js

```diff
@@ -67,7 +68,7 @@ c.explicitVoid = c.explicitThis; // error, 'void' is missing everything
 var o = {
     n: 101,
     explicitThis: function (m) {
-        return m + this.n.length; // ok, this.n: any
+        return m + this.n.length; // error, 'length' does not exist on 'number'

```

Comment:
These comments are unhelpful to future test updates, please just remove them instead of changing them in the future.